extensions-core/avro-extensions/pom.xml

```diff
@@ -35,7 +35,7 @@
 
   <properties>
     <schemarepo.version>0.1.3</schemarepo.version>
-    <confluent.version>5.5.12</confluent.version>
+    <confluent.version>6.2.12</confluent.version>
```

Comment:
any reason we can't move to a most recent 7.x?

Reply:
also, maybe we can define the confluent version as a top-level property in the main pom to avoid having to remember to update it in so many places.

Reply:
I went with closest version that has some life still in it. 
Trying to update to the latest available: 7.5.1